app/assets/javascripts/ajax_ops.js

```diff
@@ -6,11 +6,25 @@ function handle_ajax(event) {
   const createUserButton = document.getElementById('create-user');
   const userName = document.getElementById('user-username');
   const userPassword = document.getElementById('user-password');
-  const updateUserButton = document.getElementById('update-user')
-  const userID = document.getElementById('user-id')
-  const userName1 = document.getElementById('user-username1')
-  const userPassword1 = document.getElementById('user-password1')
-  const users_path = 'http://localhost:3001/api/v1/users'
+  const updateUserButton = document.getElementById('update-user');
+  const userID = document.getElementById('user-id');
+  const userName1 = document.getElementById('user-username1');
+  const userPassword1 = document.getElementById('user-password1');
+  const deleteUserButton = document.getElementById('delete-user');
+  const userID1 = document.getElementById('user-id1');
+  const listFactsButton = document.getElementById('list-facts');
+  const userID2 = document.getElementById('user-id2');
+  const createFactButton = document.getElementById('create-fact');
+  const userID3 = document.getElementById('user-id3');
+  const fact = document.getElementById('user-fact');
+  const likes = document.getElementById('user-likes');
+  const updateFactButton = document.getElementById('update-fact');
+  const userID4 = document.getElementById('user-id4');
+  const fact1 = document.getElementById('user-fact1');
+  const likes1 = document.getElementById('user-likes1');
+  const deleteFactButton = document.getElementById('delete-fact');
+  const userID5 = document.getElementById('user-id5');
+  const users_path = 'http://localhost:3001/api/v1/users';
```

Comment:
I think you want to have some variables to include the `fact content (string)`, ex.:
`const factText = document.getElementById('fact-text')`
So u can use the create/update functions on the `fact-text`.